packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -15,10 +17,13 @@ export async function findProjectRootAsync(cwd?: string): Promise<string | null>
   return projectRootDir ?? null;
 }
 
-export async function getProjectIdAsync(projectDir: string): Promise<string> {
+export const getProjectAsync = once(_getProjectAsync);
```

Comment:
Is it necessary? Do we have cases where this method is called more than once?

In the future, we will probably will add some mechanism of caching responses (mostly for credentials), but I would prefer not to use `once` for that, I think it's better for now to pass this value through some context object and maybe in some cases send request twice.

Reply:
Yes, we would have such cases if this PR got merged 😅  For example, in submission command it will be called twice.